--- stage3.orig/rootfs/etc/bash.bashrc
+++ stage3/rootfs/etc/bash.bashrc
@@ -21,6 +21,9 @@ if ! [ -n "${SUDO_USER}" -a -n "${SUDO_P
   PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
 fi
 
+alias screenlink-ro='sudo mount -o remount,ro / ; sudo mount -o remount,ro /boot/firmware'
+alias screenlink-rw='sudo mount -o remount,rw / ; sudo mount -o remount,rw /boot/firmware'
+
 # Commented out, don't overwrite xterm -T "title" -n "icontitle" by default.
 # If this is an xterm set the title to user@host:dir
 #case "$TERM" in
